client/src/omtLinkProvider.ts

```diff
@@ -81,7 +77,7 @@ function findOMTUrl(document: TextDocument, roots: Map<string, string>): Documen
                         }
                     })
 
-                    // Maak een lege Uri aan om later een relative path in te stoppen.
+                    // create an emptu\y Uri as placeholder for a relative path.
```

Comment:
typo
```suggestion
                    // create an empty Uri as placeholder for a relative path.
```

Reply:
how.... not just the change.. also me not seeing it..